pkg/build/build.go

```diff
@@ -115,6 +115,14 @@ func Clean(targetOS, targetArch, vmType, kernelDir string) error {
 	return builder.clean(kernelDir, targetArch)
 }
 
+func BuildSourcePath(params Params) (string, error) {
```

Comment:
I think it might be easier to include kernel build src into the `ImageDetails` struct instead of forcing to request the value via the function call. We'd need the value almost every time we do kernel builds in `syz-ci`.

Reply:
Done, but I caused some checkout weirdness by merging changes from the upstream branch. I'll upload the new version in a different PR.